assets/app/styles/_component-animations.less

```diff
@@ -20,3 +20,130 @@
   overflow: hidden;
   .transition(height .10s ease);
 }
+
+// Ellipsis Loader
```

Comment:
We do have ` "grunt-autoprefixer": "0.7.6",` in our `package.json` and it is running as a grunt task in `build`, `test`, etc.  We can probably skip the manual IE styles & let the task handle it for us.  It may be worth a few minutes of experimenting just to see if the generated styles are sufficient.